packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -2954,7 +2954,7 @@ Content-Type: application/octet-stream
 --abcde12345--
 ```
 
-### Payload_MultiPart_FormData_complexWithHttpPart
+### Payload_MultiPart_FormData_HttpPartTest_complexWithHttpPart
```

Comment:
nit: `Payload_MultiPart_FormData_HttpParts_x`. Also `complex` isn't a great name: what makes it complex? It appears to be that includes an array of http parts, so I would call it `Payload_MultiPart_FormData_HttpParts_Array`

Reply:
- `HttpPartTest` udpated to `HttpParts`
- "Complex" means combination of single scenarios. Array may cause confusion since it contains scenario for array of json and array of file.

Reply:
We've purposefully not used the word `complex` in `cadl-ranch` though, and this is the only case of a test with `complex`. I would prefer to have it explicitly state `JsonArrayAndFileArray` than have `complex`, because we want to be able to immediately tell exactly what we're testing from the dashboard. 

Since this is testing multiple scenarios at once, it might also mean we should split this into two tests: `_Array_Json` and `_Array_File`, unless we're explicitly testing the interplay of these 2 concepts

Reply:
We already have cases to test array of json and array of file and `complex` case is to testing the interplay of these 2 concepts. Now that name `complex` is not recommended, I update it to `jsonArrayAndFileArrayAndOthers`

Reply:
where are the json array tests? I see a model defined called `JsonArrayPartsRequest` but no test for it. I'm ok with the name, my only nit is we don't need `andOthers`, but let me go ahead and approve this pr. thanks for your work @msyyc

Reply:
Thanks for the approval.
- FileArray is only tested in `ComplexHttpPartsModelRequest` since common model can't define the valid scenario for cadl-ranch to test and only `HttpPart` can
- Removed `AndOthers` from the method name.